monix-execution/shared/src/main/scala_2.13-/monix/execution/internal/compat/package.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2014-2019 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.execution.internal
+
+import scala.collection.generic.CanBuildFrom
+import scala.collection.mutable
+
+package object compat {
+
+  private[monix] type IterableOnce[+X] = scala.collection.GenTraversableOnce[X]
+  private[monix] def toIterator[X](i: IterableOnce[X]): Iterator[X] = i.toIterator
+  private[monix] def hasDefiniteSize[X](i: IterableOnce[X]): Boolean = i.hasDefiniteSize
+
+  private[monix] type BuildFrom[-From, -A, +C] = CanBuildFrom[From, A, C]
```

Comment:
I don't know how this works, but can `private[monix]` work for this type alias, given that it is exposed in public signatures? Isn't the compiler exposing any warnings?

Also you're exposing both `monix.execution.internal.compat` and `monix.execution.compat.internal`, better to choose one.

Reply:
Forgot to delete this file after changing to `monix.execution.compat.internal`